### ISSUE TITLE
build bootloader using armgcc

### ISSUE THREAD

**Issue by rgrover** [mbedmicro/BLE_API#27]
post instructions on building bootloader using armgcc.

**Comment by jrobeson** [comment]
any update on this?

**Comment by rgrover** [comment]
No. There are community pages on mbed.org (team Bluetooth Low Energy) to describe the process of building the bootloader. We haven't had the time to invest in creating build instructions using gcc.

**Comment by rainierwolfcastle** [comment]
http://jira-dev0.cambridge.arm.com/browse/IOTSFW-865

**Comment by rainierwolfcastle** [comment]
ARM Internal Ref: IOTSFW-1036
